compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -227,6 +240,10 @@ impl MetaItem {
     pub fn is_value_str(&self) -> bool {
         self.value_str().is_some()
     }
+
+    pub fn name_value_literal_span(&self) -> Option<Span> {
```

Comment:
Not sure if this function is worth it

Reply:
Not sure either. I had the need for it in rustdoc but then removed it. So as you prefer.

Reply:
let's leave it out and just do its body manually at its use sites

Reply:
Removed! :)